src/Pages/StatusPage/Create/Components/Tabs/index.jsx

```
@@ -26,28 +27,20 @@ const Tabs = ({
-					{TAB_LIST.map((tab, idx) => {
-						return (
-							<Tab
-								key={tab}
-								label={TAB_LIST[idx]}
-								value={TAB_LIST[idx]}
-							/>
-						);
-					})}
-				</TabList>
-			</Box>
+				{TAB_LIST.map((tabLabel, idx) => (
```

ESLint

[error] 'idx' is defined but never used.

(no-unused-vars)